docs/index.rst

```diff
@@ -8,12 +8,31 @@
    :class:`multiprocessing.Pool` or :class:`multiprocessing.Process` invokes
    :func:`os.fork`.
 
+Guides
+------
+.. toctree::
+  :maxdepth: 2
+
+  acl_guide
+  generation_metageneration
+  retry_timeout
+
 API Reference
 -------------
 .. toctree::
   :maxdepth: 2
 
-  storage/modules
+  storage/acl
+  storage/batch
+  storage/blob
+  storage/bucket
+  storage/client
+  storage/constants
```

Comment:
Constants are not rendering on c.g.c. Let's discuss further in our call

Reply:
Closing the loop - this is a known issue and I'll look into it separately.